tests/unit/ui/components/TestSearchInput.js

```diff
@@ -1,6 +1,7 @@
 /* global window */
 import React from 'react';
 import { renderIntoDocument, Simulate } from 'react-addons-test-utils';
+import sinon from 'sinon';
```

Comment:
You shouldn't need to explicitly import `sinon`; it's loaded into our tests by default.